src/accountScreen/accountScreen.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import './accountScreen.css'
+import SearchBar from 'material-ui-search-bar'
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
+
+
+
+const ChatCard = function(props) {
+  if(props.selected){

```

Comment:
Always put spaces around operators ( = + - * / ), and after commas:

Reply:
Done. Noted